### PR TITLE
User profile  stylez

### LINKED ISSUE #1881: [User Profile] Style the user profile page.

### LINKED ISSUE #1899: [User Profile] User email is showing up as logged in user and not account user
https://trello.com/c/ou9Ggl7t/1113-user-profile-bug

### PR DESCRIPTION
**Pull request by weerd**
Resolves  #1881
Resolves #1899

@DFurnes Once the design stuff reaches final approval I'll work towards moving the patterns established over to Neue; for the time being they're in flux!

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/scss/helpers/_mixins.scss [hunk 1]
```diff
@@ -152,3 +152,15 @@
...
+
+
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// CLEARFIX
+//
+// Contain floats: h5bp.com/q
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.clearfix:before, .clearfix:after { content: ""; display: table; }
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/helpers/_mixins.scss:164]: Let's use the Bourbon [clearfix mixin](http://bourbon.io/docs/#clearfix) for this.

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/campaign-block.tpl.php [hunk 2]
```diff
@@ -9,11 +9,16 @@
...
  *   - [path]: URL path for campaign node (string).
+ *   - [pretty_path]: Pretty URL path for campaign node (string).
  */
 ?>
 
-<article class="campaign -teaser" id="campaign-<?php print $vars['nid']; ?>">
-  <h1><?php print l($vars['title'], $vars['path']); ?></h1>
-  <p><?php print $vars['call_to_action']; ?></p>
-  <img alt="<?php print $vars['title']; ?>" src="<?php print $vars['image']; ?>" />
+<article class="tile tile--campaign" id="campaign-<?php print $vars['nid']; ?>">
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/campaign-block.tpl.php]: This looks good. Let's use this same markup for [the campaign thumbnails](https://github.com/DoSomething/dosomething/blob/dev/lib/themes/dosomething/paraneue_dosomething/templates/home/partials/thumbnail.tpl.php) I had built for the homepage and for the [Campaign Finder template](https://github.com/DoSomething/dosomething/blob/dev/lib/themes/dosomething/paraneue_dosomething/js/finder/templates/campaign.tpl.html) so we don't have two different implementations of the same pattern.

**Comment by DFurnes** [comment]
:sun_with_face:

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_user.scss [hunk 3]
```diff
@@ -1,3 +1,206 @@
...
+// @TODO: Ideally, make this universal... no scenarios where the nav shouldn't position abs on top of header.
+.page-user {
+  @include wrapper--nav-white;
+  @include wrapper--nav-floating;
+}
+
+
+
+// @TODO: Move to Neue.
+.tile { background-color: fuchsia;
```
Comment (barryclark) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_user.scss]: Not sure we want to keep the fuchsia :)

File: lib/themes/dosomething/paraneue_dosomething/scss/helpers/_mixins.scss (hunk 1 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/helpers/_mixins.scss:164]: I wanted to chat about this. I feel there are cases where you want to be able to add the clearfix as a class and other cases where it makes sense to be a mixin. If you always add it in as a mixin it ends up bloating your css w/ the same code over and over...

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/helpers/_mixins.scss:164]: Gzipping should solve the file size concerns. We can always keep both options in if you think there's situations where you'd want a more situational clearfix I guess.

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/helpers/_mixins.scss:164]: I always forget the gzipping point ;) I'd still like to pop it into Neue so it's available as an option like you mentioned. I can do that during the cleanup phase along with the `.tile { }` stuff I need to move over.

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/helpers/_mixins.scss:164]: :+1:

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/campaign-block.tpl.php (hunk 2 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/campaign-block.tpl.php]: Cool. I can combine them a bit.

**Comment by DFurnes** [comment]
Looks HOT. :fire:

**Pull request merged**